becquerel/core/spectrum.py

```diff
@@ -581,7 +581,7 @@ def from_file(cls, infilename):
 
     @classmethod
     def from_listmode(cls, listmode_data, bins=None, xmin=None, xmax=None,
-                      **kwargs):
+                      is_cal=False, **kwargs):
```

Comment:
Please document the new kwarg and update the docstring to refer not only to `bin_edges_raw`.